internal/cli/auth/register.go

```diff
@@ -22,17 +22,15 @@ import (
 	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
 	"github.com/mongodb/mongodb-atlas-cli/internal/cli/require"
 	"github.com/mongodb/mongodb-atlas-cli/internal/config"
+	"github.com/mongodb/mongodb-atlas-cli/internal/oauth"
 	"github.com/spf13/cobra"
 	"go.mongodb.org/atlas/auth"
 	atlas "go.mongodb.org/atlas/mongodbatlas"
 )
 
-//go:generate mockgen -destination=../../mocks/mock_register.go -package=mocks github.com/mongodb/mongodb-atlas-cli/internal/cli/auth RegisterFlow
+//go:generate mockgen -destination=../../mocks/mock_register.go -package=mocks github.com/mongodb/mongodb-atlas-cli/internal/cli/auth RegisterAuthenticator,RegisterFlow
```

Comment:
the fact that we are generating mocks from this file and it does not live in the store looks suspicious, maybe a refactoring to move these to the Store would make more sense

Reply:
good point about having the auth calls included in the store. I will do more research here and I might include it in a list of nice to have topics for future refactoring.